### ISSUE TITLE
URL to set Github Repo Website for Resource Kit repositories

### ISSUE THREAD

**Issue by PlagueHO** [PowerShell/DscResources#338]
After seeing @johlju make a request to get the SQLServerDsc resource repo description set, I started making the requests to get all the others set as well (sorry @kwirkykat  :grin: ). But it crossed my mind: Should we also be setting the website for these as well?

I notice that DSCResources is set to:
http://blogs.msdn.com/b/powershell/

But another alternative for the individual resources could be:
https://blogs.msdn.microsoft.com/powershell/tag/dsc-resource-kit/

Is there a recommendation or suggestion from the PowerShell team for this?

**Comment by kwirkykat** [comment]
@PlagueHO For now, I set the website to the central DscResources repo since it has all our documentation for support, naming, contributing, etc. But either one of those links you suggested would be fine too. We could also link to the module on the PowerShell Gallery. Not sure which is best yet, but set to DscResources until someone has an argument otherwise.

**Comment by PlagueHO** [comment]
All good! Thanks @kwirkykat - I don't really have much of a preference either way, just thought it was worth raising. I'm happy to just leave this issue here for some discussion - no need to rush on making a call on this IMHO.

**Comment by johlju** [comment]
I agree with @kwirkykat here, for me it feels correct having it set to DscResources.

**Comment by johlju** [comment]
@PlagueHO What do you say? Should we close this issue? Or should we change the default?
